app/controllers/files_controller.rb

```diff
@@ -9,6 +9,15 @@ def index
     raise ArgumentError, 'Missing file parameter' if filename.blank?
 
     @available_in_workspace = Dor::Services::Client.object(params[:item_id]).files.list.include?(filename)
+    @has_been_accessioned = Dor::Config.workflow.client.lifecycle('dor', params[:item_id], 'accessioned')
+
+    if @has_been_accessioned
+      begin
+        @last_accessioned_version = Preservation::Client.objects.current_version(params[:item_id])
+      rescue Preservation::Client::NotFoundError
```

Comment:
Worth considering a block-level `rescue` here instead of the explicit `begin`/`end`? Don't care **much** about it but it'd drop 2LOC :laughing:

Reply:
Begin and end define a block, so this is a block-level rescue.  I'm not sure I understand what you're after.

Reply:
Oh, my bad. I misread the `if` block here as a "block" block. :tea: